app/assets/javascripts/lit/lit_frontend.js

```diff
@@ -11,11 +11,15 @@
     var meta = $('meta[name="lit-url-base"]');
     if(meta.length > 0){
       getLocalizationPath($this, meta);
-      //replaceWithForm(e.currentTarget, value, update_path)
+    }
+    else
+    {
+      console.error('cannot lit base url');
+      alert('cannot lit base url');
```

Comment:
I'd leave only `console.log` with proper error message ;)

Reply:
You mean `console.error`? Ok I'll remove the `alert()`

Reply:
Done